addons/survey/static/src/js/survey_image_zoomer.js

```diff
@@ -0,0 +1,121 @@
+/** @odoo-module */
+
+import publicWidget from 'web.public.widget';
+
+export const surveyImageZoomer = publicWidget.Widget.extend({
+    template: 'survey.survey_image_zoomer',
+    xmlDependencies: ['/survey/static/src/xml/survey_image_zoomer_templates.xml'],
+    events: {
+        'wheel .o_survey_img_zoom_image': '_onImgScroll',
+        'click': '_onZoomerClick',
+        'click .o_survey_img_zoom_in_btn': '_onZoomInClick',
+        'click .o_survey_img_zoom_out_btn': '_onZoomOutClick',
+    },
+    /**
+     * @override
+     */
+    init(params) {
+        this.zoomImageScale = 1;
+        // The image is needed to render the template survey_image_zoom.
+        this.sourceImage = params.sourceImage;
+        this._super(... arguments);
+    },
+    /**
+     * Open a transparent modal displaying the survey choice image.
+     * @override
+     */
+    async start() {
+        const superResult = await this._super(...arguments);
+        this.$el.modal({
+            keyboard: true,
+            backdrop: false,
+        });
+        // Prevent having hidden modal in the view & tell the form the modal closed.
+        this.$el.on('hidden.bs.modal', () => {
+            this.trigger('close_survey_img_zoom');
+            this.destroy();
+        });
+        this.$el.modal('show');
+        return superResult;
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * Zoom in/out image on scrolling
+     *
+     * @private
+     * @param {MouseEvent} e
```

Comment:
I know "clicks" are instances of "MouseEvent", not sure about the mousewheel though, I'm guessing you checked?